geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -709,11 +722,31 @@ private boolean isStartable() {
    * @see org.apache.geode.distributed.AbstractLauncher.Status#ONLINE
    * @see org.apache.geode.distributed.AbstractLauncher.Status#STARTING
    */
-  @SuppressWarnings("deprecation")
   public LocatorState start() {
     if (isStartable()) {
       INSTANCE.compareAndSet(null, this);
 
+      boolean serializationFilterConfigured = false;
+      if (isJavaVersionAtLeast(JAVA_1_8) && isJavaVersionAtMost(JAVA_1_8)) {
```

Comment:
shouldn't this use the new "isJavaVersion" method you added?

Reply:
We have removed `isJavaVersion`. And keep `isJavaVersionAtLeast` and `isJavaVersionAtMost`, which are more convenient to handle multiple Java versions, instead of a specific version.